scripts/meterpreter/multi_console_command.rb

```diff
@@ -25,18 +26,28 @@
 commands = nil
 script = []
 help = 0
+silence = 0
 
 ################## Function Declarations ##################
 # Function for running a list of commands stored in a array, returs string
-def list_con_exec(cmdlst)
+def list_con_exec(cmdlst, silence)
   print_status("Running Command List ...")
```

Comment:
A quick look through the code said this function is only called once in framework, but from my perspective, it would still make sense to use a default value for the second parameter so that it maintains the previous behavior if someone calls this method the old way with only one parameter.  What are your thoughts on changing this to:
```def list_con_exec(cmdlst, silence = false)```  ?